### DISCUSSION TITLE
Supabase Function hook not triggering on Insert (resolved)

### DISCUSSION THREAD

**Discussion by AlexanderCryptoOwl** [supabase/supabase#5664] [Questions] [answered]
Hello there :),

I have a simple function hook that triggers an HTTPS request on a table insert. The request trigger my Cloud function but looking at GC logs nothing is coming?

<img width="1001" alt="image" src="https://user-images.githubusercontent.com/99205909/155698222-2debe345-161e-4e25-b15b-a7144af92088.png">

Can I see in Supabase logs if something is happening? Couldn't see anything

I know not offering much contexts but maybe got tips?

Cheers!

**Comment by GaryAustin1** [comment, marked as answer]
Did you resolve this?  If not, and if your profiles table is being inserted by an auth table trigger during user signup, can you try insert trigger on another table and see if it works? 
Just a wild guess that an insert trigger on an auth insert, might act differently based on path or privileges.

**Reply by AlexanderCryptoOwl** [reply]
Hi thank you for replying! I actually made it work :)

**Reply by navandstokes** [reply]
What was your fix? I'm encountering the same issue.

**Reply by AlexanderCryptoOwl** [reply]
Haha, it wasn't a Supabase issue for me but a Google Cloud function permission problem and they didn't log that so that's why couldn't see it.
